modules/editor/inserter/index.js

```diff
@@ -0,0 +1,22 @@
+const h = wp.element.createElement;
```

Comment:
I have a personal distaste for single letter vars. Though I could compromise on `el` ;)